test/new_relic/agent/method_tracer_params_test.rb

```diff
@@ -1,178 +0,0 @@
-# encoding: utf-8
```

Comment:
Why was this file removed? I think this is still the type of thing we'll want to be testing, especially because of kwarg reasons?

Reply:
I have mixed feelings about this.

For one thing, this file is testing Ruby functionality and not our actual agent code. For another, it's asserting deprecation warnings which don't fire because we're using `ruby2_keywords` as intended.

I think it's a good idea to test different parameter formats to verify the delegation works, but not necessarily at this level of granularity. We can add this to our existing method tracer tests and let the CI test against different Ruby versions.

Reply:
i think that makes sense. My main concern related to this is making sure we are forwarding the arguments correctly, no matter if someone is using old style args, kwargs, etc, to make sure the arguments are exactly as the user expects. If we add some tests elsewhere that can check for that, I think that sounds good to me.